WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -168,6 +168,11 @@ + (void)setShouldWaitForQuiescence:(BOOL)value
 
 + (NSUInteger)mjpegServerFramerate
 {
+  NSUInteger frameRate = self.mjpegServerFrameRateFromArguments;
+  if (frameRate != NSNotFound) {
```

Comment:
Do we want to ignore setting the framerate from settings if it was set via the command line?

Reply:
yeah, they are kind of exclusive. well, why not? (it's pretty much same logic as as with ```--port``` and ```--mjpeg-server-port```).

if later on decided to override this value with request from settings API request, then it needs a bit different logic at different level. (but i doubt that someone would use this args / options in such a way). at least it's hard to find reasoning behind this - setting frame rate from args, and then overriding from settings — pretty much WDA sessions are not that long - like one-two-ten tests.

Reply:
what could be the use case for changing frame rate between tests anyway?